_sass/base.scss

```diff
@@ -51,3 +51,7 @@ img {
   width: 100%;
   height: auto;
 }
+
+.text-italic {
```

Comment:
Suggest that we prefer to use `<em>` in most cases and then sparingly apply style semantically rather than functionally. For this case having a `quotation` class (which applies this styling) might make sense? Makes it more obvious when adding a new quotation that it should have the same style.